buildspecs/linux_ppc-64_cmprssptrs_le.spec

```diff
@@ -129,7 +129,10 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<flag id="build_java8" value="true"/>
 		<flag id="build_java9" value="false"/>
 		<flag id="build_product" value="true"/>
+		<flag id="build_openj9" value="true"/>
```

Comment:
I think this and the next are  obsolete properties that could be removed from all specs? These values aren't set in the non-cmprssptrs spec, so I'm guessing they aren't used. Anyway, if we make changes for this, it should be done separately.

Reply:
will clean up in a different PR